### DISCUSSION TITLE
Running tests in a directory parallely

### DISCUSSION THREAD

**Discussion by EshaanAgg** [vitest-dev/vitest#2891] [Q&A] [answered]
I am using `Vitest` as the testing framework for my project.

I have a directory called `canRunInParallel` which contains multiple test files, like `A.spec.ts`, `B.spec.ts` .....  `Z.spec.ts`. Since this directory contains multiple test files, and none of the tests have a race condition among themselves, I want to configure `Vitest` to run all the tests files in this directory concurrently, so that I can reduce the testing time of my application. 

Does `Vitest` support this?

**Comment by sheremet-va** [comment, marked as answer]
Vitest doesn't support this
